src/class/SimpleData.ts

```diff
@@ -34,27 +33,49 @@ import mergeItems_ from "../methods/mergeItems.js"
 import saveData_ from "../methods/saveData.js"
 import saveChart_ from "../methods/saveChart.js"
 import saveCustomChart_ from "../methods/saveCustomChart.js"
-import { SimpleDataItem, Options, defaultOptions } from "../types/SimpleData.types.js"
-import getParametersAndOptions from "../helpers/getParametersAndOptions.js"
-import logInfos from "../helpers/logInfos.js"
+import checkKeys from "../helpers/checkKeys.js"
+import logCall from "../helpers/logCall.js"
+import checkEnvironment from "../helpers/checkEnvironment.js"
+import { SimpleDataItem } from "../types/SimpleData.types"
 
 
 export default class SimpleData {
 
     _data: SimpleDataItem[]
     _keys: string[]
-    _defaultOptions: Options
+    _environment: "nodejs" | "webBrowser"
+    // Logging 
+    _verbose: boolean
+    _logParameters: boolean
+    _nbTableItemsToLog: number
+
+    constructor(
+        data: SimpleDataItem[], 
+    {
+        verbose = false,
+        logParameters = false,
+        nbTableItemsToLog = 5
+    }: {
+        verbose?: boolean,
+        logParameters?: boolean,
+        nbTableItemsToLog?: number
+    } = {}) {
+        checkKeys(data)
+
+        this._data = data
+        this._keys = Object.keys(data[0])
+
+        this._verbose = verbose
```

Comment:
Maybe add a get / set for each options. So the user can easily change them for all methods?

Reply:
I have set them as public attribute now, so they can be modified by doing `simpleData.verbose = true`